### ISSUE TITLE
Upload a file with spaces in the path

### ISSUE THREAD

**Issue by gitter-me** [Huachao/vscode-restclient#376]
- VSCode Version: 1.35.1
- OS Version: Win10 1809
- REST Client Version: 0.21.3

Steps to Reproduce:

I tried to upload a file with spaces on windows, not getting it to work:

```
POST http://localhost:1337/api/v1/someendpoint HTTP/1.1
content-type: image/jpeg

< C:\Users\Username\Documents\Path with spaces\Filename with spaces.jpg
```
I tried putting the path in single and double quotes, tried to use a custom variable, and also tried to replace the spaces in the string with %20

**Comment by Huachao** [comment]
@gitter-me nice catch, I have fixed the issue, and now you can keep the original file path with spaces as you did(no escape is needed). And I will publish this fix in next release.

**Comment by Huachao** [comment]
@gitter-me I have fixed this in latest version 0.22.0, and you can try this fix, and you just need to keep the original whitespaces in the file path
